### PR TITLE
Cards vue C-E

### PR DESCRIPTION
**Pull request by chosta**
Just to keep the momentum going

* Added 20 more cards (skipping some as they need some additional logic)
* Added jovian, venus tags and = sign

### CODE REVIEW

File: src/cards/promo/CometAiming.ts [hunk 1]
```diff
@@ -97,4 +99,18 @@ export class CometAiming implements IActionCard, IProjectCard, IResourceCard {
...
+      cardNumber: 'X15',
+      renderData: CardRenderer.builder((b) => {
+        b.effectBox((eb) => {
+          eb.titanium(1).startAction.asteroids(1).asterix();
+          eb.description('Action: Spend 1 titanium to add 1 asteroid resource to ANY CARD');
+        }).br;
+        b.or().br;
+        b.effectBox((eb) => {
+          eb.asteroids(1).startAction.oceans(1);
+          eb.description('Action: remove 1 asteroid here to place an ocean');
```
Comment (nwai90) [inline src/cards/promo/CometAiming.ts:112]: Minor: The "remove" in `remove 1 asteroid` should be capitalized

File: src/cards/render/CardRenderItemType.ts [hunk 2]
```diff
@@ -16,6 +16,8 @@ export enum CardRenderItemType {
   ANIMALS = 'animals',
   EVENT = 'event',
   SPACE = 'space',
+  JOVIAN = 'jovian',
+  VENUS_TAG = 'venus tag',
```
Comment (nwai90) [inline src/cards/render/CardRenderItemType.ts:20]: Minor: Perhaps `venus_tag` for consistency with TRADE_DISCOUNT, TRADE_FLEET and PARTY_LEADERS

**Review by nwai90 — APPROVED** [review]

File: src/cards/Decomposers.ts [hunk 3]
```diff
@@ -27,5 +31,12 @@ export class Decomposers implements IProjectCard, IResourceCard {
     public play() {
       return undefined;
     }
+    public metadata: CardMetadata = {
+      cardNumber: '131',
+      requirements: CardRequirements.builder((b) => b.oxygen(3)),
+      description: 'Requires 3% oxygen. 1 VP per 3 Microbes on this card',
+      renderData: CardRenderer.builder((b) => b.effectBox((be) => be.animals(1).played.slash().plants(1).played.slash().microbes(1).played.slash().startEffect.microbes(1).description('Effect: When you play an Animal, Plant, or Microbe tag, including this, add a Microbe to this card'))),
```
Comment (kberg) [inline src/cards/Decomposers.ts:38]: nit: Mind making this four or five lines?

File: src/cards/DeimosDown.ts [hunk 4]
```diff
@@ -33,4 +35,13 @@ export class DeimosDown implements IProjectCard {
       player.steel += 4;
       return undefined;
     }
+    public metadata: CardMetadata = {
+      cardNumber: '039',
+      description: 'Raise temperature 3 steps and gain 4 steel. Remove up to 8 Plants from any player',
+      renderData: CardRenderer.builder((b) => {
+        b.temperature(3).br;
+        b.steel(4).br;
+        b.minus().plants(-8).any;
```
Comment (kberg) [inline src/cards/DeimosDown.ts:44]: So in this case you have both the minus and the -8. What gives?

File: src/cards/ares/DesperateMeasures.ts [hunk 5]
```diff
@@ -40,4 +42,12 @@ export class DesperateMeasures implements IProjectCard {
     public getVictoryPoints() {
       return -2;
     }
+    public metadata: CardMetadata = {
+      cardNumber: 'A04',
+      description: 'Effect: Place a bronze cube on a dust storm tile and raise oxygen 1 step, or place a bronze cube on an erosion tile and raise the temperature 1 step. The hazard tile with the bronze cube cannot be removed',
+      renderData: CardRenderer.builder((b) => {
+        b.temperature(1).slash().oxygen(1);
```
Comment (kberg) [inline src/cards/ares/DesperateMeasures.ts:49]: Would you please add a TODO to visualize the bronze cube?

**Review by kberg — APPROVED** [review]
After addressing @nwai90's comments, feel free to merge. This looks great!

File: src/cards/DeimosDown.ts (hunk 4 above)
Comment (chosta) [reply src/cards/DeimosDown.ts:44]: I'll look for an abstraction of this behavior in future PR. You have a point

File: src/cards/Decomposers.ts (hunk 3 above)
Comment (chosta) [reply src/cards/Decomposers.ts:38]: y, it's a bit cluttered

**Pull request merged**